spec/lib/html_cleaner_spec.rb

```diff
@@ -6,7 +6,7 @@
   include HtmlCleaner
 
   describe "TagStack" do
-    let(:stack) { TagStack.new }
+    let(:stack) { HtmlCleaner::TagStack.new }
```

Comment:
How did this break without `HtmlCleaner::`? I like the change but I'm curious.

Reply:
It produced a whole stream of errors like this one:
```
  1) HtmlCleaner TagStack inside paragraph? should return false
     Failure/Error: let(:stack) { TagStack.new }

     NameError:
       uninitialized constant TagStack
     # ./spec/lib/html_cleaner_spec.rb:9:in `block (3 levels) in <top (required)>'
     # ./spec/lib/html_cleaner_spec.rb:13:in `block (4 levels) in <top (required)>'
```
I think it's because nested `describe`/`context` blocks in RSpec tests actually become nested classes. When `include HtmlCleaner` wasn't nested in a module/class, it was included in all classes/modules, including RSpec tests. But now it's only included in the root `describe HtmlCleaner` block, not inside the nested `describe TagStack` block, so it's looking for the `TagStack` class in the wrong nesting.